### PR TITLE
Update to work with 1.18.2

### LINKED ISSUE #1: 1.18.1 support?
the title says everything

### PR DESCRIPTION
**Pull request by Douile**
Closes #1

### CODE REVIEW

**Comment by Nekoshenka** [comment]
How to install ur 1.18.2 patch into a mod?

File: build.gradle [hunk 1]
```diff
@@ -17,8 +17,8 @@ dependencies {
 	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
 
 	// Fabric API. This is technically optional, but you probably want it anyway.
-	modImplementation "net.fabricmc.fabric-api:fabric-api-base:${project.api_base_version}"
-	include "net.fabricmc.fabric-api:fabric-api-base:${project.api_base_version}"
+	modImplementation "net.fabricmc.fabric-api:fabric-api:${project.api_base_version}"
+	include "net.fabricmc.fabric-api:fabric-api:${project.api_base_version}"
```
Comment (wagyourtail) [inline build.gradle:21]: This is not correct. i was including specific subsections of fabric api, and this is the whole thing

**Review by wagyourtail — CHANGES_REQUESTED** [review]

**Comment by Nekoshenka** [comment]
@wagyourtail can u answer me how i can get jar file (which one i put in mods directory) after wrapping source file by gradle (7.5.1 version i use) 
Help pls... i'm quite new in this programs...

File: build.gradle (hunk 1 above)
Comment (Douile) [reply build.gradle:21]: This was quick and dirty fix to get to build, I believe there was some problem with just using fabric-api-base. lmgbty

**Comment by Douile** [comment]
> @wagyourtail can u answer me how i can get jar file (which one i put in mods directory) after wrapping source file by gradle (7.5.1 version i use) Help pls... i'm quite new in this programs...

https://fabricmc.net/wiki/documentation:fabric_loom?s[]=build

`./gradlew remapJar`

**Comment by Nekoshenka** [comment]
> > @wagyourtail can u answer me how i can get jar file (which one i put in mods directory) after wrapping source file by gradle (7.5.1 version i use) Help pls... i'm quite new in this programs...
> 
> [https://fabricmc.net/wiki/documentation:fabric_loom?s[]=build](https://fabricmc.net/wiki/documentation:fabric_loom?s%5B%5D=build)
> 
> `./gradlew remapJar`

I’ve been sitting for the second day and I don’t understand how gradle works and how I can get the desired mod... Can I ask you to send me the finished mod... Forgive me for impudence, but I really can’t understand reading almost all the tutorials, I either have something either it crashes or nothing happens.
(Also command "gradlew" in unkown but "gradle" work,

**Pull request closed without merging**